src/lib/utils.ts

```diff
@@ -38,3 +38,15 @@ export const camelCase = (str: string) => {
 
   return camelCase
 }
+
+export const sendAdminNewDeal = async (dealId: string) => {
```

Comment:
This doesn't need to call an API endpoint. This logic can be called directly inside of the action instead. This function should be deleted from `utils.ts`